### ISSUE TITLE
Add IP Allow List to LoadBalancedWeb Service Pattern

### ISSUE THREAD

**Issue by kohidave** [aws/copilot-cli#1234]
There are lots of occasions where you might want to limit the folks who can access their load balanced web service based on IP address range.

This is something you can do using [Source IP Conditions](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-listeners.html#source-ip-conditions) in our listener configuration. 

We might consider updating our manifest to look like:

```yaml

name: my-restricted-service
# The "architecture" of the service you're running.
type: Load Balanced Web Service

image:
  # Path to your service's Dockerfile.
  build:./Dockerfile
  # Port exposed through your container to route traffic to it.
  port: 3030

http:
  # Requests to this path will be forwarded to your service. 
  # To match all requests you can use the "/" path. 
  path: /
  # Only allow callers from this list of IP addresses
  allow-list:
    - 203.0.113.0/24
```

**Comment by softprops** [comment]
Alternatively you might control ingress with an lb [SecurityGroup](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-loadbalancer.html#cfn-elasticloadbalancingv2-loadbalancer-securitygroups) inbound rule list

**Comment by efekarakus** [comment]
This was released in v0.6.0 https://github.com/aws/copilot-cli/releases/tag/v0.6.0 happy building! 🚀
